cfme/tests/configure/test_landing_page.py

```diff
@@ -105,13 +102,18 @@
     'Red Hat Access Insights']
 
 
-def set_landing_page(value, appliance):
+@pytest.yield_fixture(scope='module')
```

Comment:
Q: why yield_fixture ?

And I doubt you need such function at all. You can just add such definition to test or to both fixtures.

Reply:
It was repetitive so I though of using it, moreover if I add some more test case related to rbac user I can directly use it.

Reply:
@anurag03, honestly I don't see any benefit of wrapping one row in function/fixture. It doesn't bring any value to you. you just put another fixture into test definition instead of defining that variable in test.